backend/api/api_request_classes.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import Field
+
+from backend.common import models
+
+
+class CreateUserRequest(models.User):
+    password: str = Field(min_length=8, max_length=128)
+
+
+class CreateGroupRequest(models.Group):
```

Comment:
As said before, this class is not required. You can use the parent class for the request